venv/lib/python3.9/site-packages/pip/_internal/commands/check.py

```
@@ -0,0 +1,48 @@
+from pip._internal.utils.misc import write_output
+
+logger = logging.getLogger(__name__)
+
+
+class CheckCommand(Command):
+    """Verify installed packages have compatible dependencies."""
+
+    usage = """
+      %prog [options]"""
+
+    def run(self, options, args):
```

Ruff (0.14.8)

Unused method argument: `options`

(ARG002)